dynamicEDT3D/include/dynamicEDT3D/dynamicEDTOctomap.hxx

```diff
@@ -35,12 +35,16 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <dynamicEDT3D/dynamicEDTOctomap.h>
+#include "dynamicEDT3D/dynamicEDTOctomap.h"
```

Comment:
I think this line should be removed - the .hxx is included by the header and the .hxx should not be directly included anyways.